src/main/java/webserver/RequestHandler.java

```diff
@@ -22,10 +26,10 @@ public void run() {
         try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
             // TODO 사용자 요청에 대한 처리는 이 곳에 구현하면 된다.
             RequestManager requestManager = new RequestManager();
-            HttpRequest httpRequest = requestManager.parseInputStream(in);
-
             ResponseManager responseManager = new ResponseManager();
-            HttpResponse httpResponse = responseManager.createResponse(httpRequest);
+
+            HttpRequest httpRequest = requestManager.parseInputStream(in);
+            HttpResponse httpResponse = responseManager.findHandler(httpRequest);
             httpResponse.send(out);
```

Comment:
```suggestion
            HttpRequest httpRequest = requestManager.parseInputStream(in);
            Handler handler = responseManager.findHandler(httpRequest);
            HttpResponse httpResponse = hadler.createResponse();
            httpResponse .response(out);
```
메소드 이름만 보고 생각해 보면 이런 정도의 흐름이 자연스러워 보입니다.